dev-env/src/key-manager/index.ts

```diff
@@ -1,15 +1,18 @@
+/**
+ * NOTE
+ * This key manager service is not meant for production use.
+ * It's specifically designed for the dev env.
+ */
+
 import http from 'http'
 import express from 'express'
 import cors from 'cors'
-import axios from 'axios'
 import * as uint8arrays from 'uint8arrays'
 import * as crypto from '@adxp/crypto'
 
 import KeyManagerDb from './db.js'
 import { formatDidWeb } from './did.js'
 
-const DID_SERVER = 'localhost:2582'
-
 export const runServer = (db: KeyManagerDb, port: number): http.Server => {
```

Comment:
I think every service's `runServer` method is a bit different. 
Some don't return anything.
Some return the `http.Server`
Some console.log the "listening" message
The DidWebServer returns a class that let's you interact with the server without making http calls.
(this incongruity is mostly my fault 😅)

What's your general preference on these? & we can work to make them sync up

Reply:
I got them all mostly equivalent, there's just one (the did:web server I think) that isn't. We can do another pass later to unify more

Reply:
One thing I did do was move all the console.logs outside of the APIs

Reply:
Ah I guess i missed the moving the `console.logs` out.

Yeah i refactored the DidWebServer last week, but left it as a class you can interact with without http for the tests/etc. Wasn't sure if we'd want that when working in dev-env 🤷‍♂️